specification/provisioningservices/resource-manager/Microsoft.Devices/2017-08-21-preview/iotdps.json

```diff
@@ -0,0 +1,1600 @@
+						"type": "string"
+					}
+				},
+				"features": {
+					"enum": ["None", "DeviceManagement"],
+					"type": "string",
+					"x-ms-enum": {
+						"name": "Capabilities",
+						"modelAsString": "True"
+					}
+				}
+			}
+		},
+		"CertificateResponse": {
+			"type": "object",
+			"properties": {
+				"name": {
+					"type": "string"
+				},
+				"etag": {
+					"type": "string"
+				},
+				"subject": {
+					"type": "string"
+				},
+				"expiry": {
+					"format": "date-time",
+					"type": "string"
+				},
+				"thumbprint": {
+					"type": "string"
+				},
+				"isVerified": {
+					"type": "boolean"
+				},
+				"created": {
+					"format": "date-time",
+					"type": "string"
+				},
+				"updated": {
+					"format": "date-time",
+					"type": "string"
+				}
+			}
+		},
+		"IotDpsSkuInfo": {
+			"type": "object",
+			"properties": {
+				"name": {
+					"enum": ["S1"],
+					"type": "string",
+					"x-ms-enum": {
+						"name": "IotDpsSku",
+						"modelAsString": "True"
+					}
+				},
+				"tier": {
+					"type": "string",
+					"readOnly": true
+				},
+				"capacity": {
+					"format": "int64",
+					"type": "integer"
+				}
+			}
+		},
+		"ProvisioningServiceDescription": {
+			"type": "object",
+			"properties": {
+				"etag": {
+					"type": "string"
+				},
+				"name": {
+					"type": "string",
+					"readOnly": "true"
+				},
+				"location": {
+					"type": "string"
+				},
+				"properties": {
+					"$ref": "#/definitions/IotDpsPropertiesDescription"
+				},
+				"type": {
+					"type": "string",
+					"readOnly": true
+				},
+				"id": {
+					"type": "string",
+					"readOnly": true
+				},
+				"tags": {
+					"type": "object",
+					"additionalProperties": {
+						"type": "string"
+					}
+				}
+			}
+		},
+		"IotDpsPropertiesDescription": {
+			"type": "object",
+			"properties": {
+				"state": {
+					"enum": ["Activating", "Active", "Deleting", "Deleted", "ActivationFailed", "DeletionFailed", "Transitioning", "Suspending", "Suspended", "Resuming", "FailingOver", "FailoverFailed"],
+					"type": "string",
+					"x-ms-enum": {
+						"name": "Nullable`1",
+						"modelAsString": "True"
+					}
+				},
+				"provisioningState": {
+					"type": "string"
+				},
+				"iotHubs": {
+					"type": "array",
+					"items": {
+						"$ref": "#/definitions/IotHubDefinitionDescription"
+					}
+				},
+				"allocationPolicy": {
+					"enum": ["Hashed", "GeoLatency", "Static"],
+					"type": "string",
+					"x-ms-enum": {
+						"name": "AllocationPolicy",
+						"modelAsString": "True"
+					}
+				},
+				"serviceOperationsHostName": {
+					"type": "string",
+					"readOnly": "true"
+				},
+				"deviceProvisioningHostName": {
+					"type": "string",
+					"readOnly": "true"
+				},
+				"idScope": {
+					"type": "string"
+				},
+				"authorizationPolicies": {
+					"type": "array",
+					"items": {
+						"$ref": "#/definitions/SharedAccessSignatureAuthorizationRule[AccessRightsDescription]"
```

Comment:
I dont think this will work SharedAccessSignatureAuthorizationRule[AccessRightsDescription]. not sure what it means but @feathercowboy can provide better input here.

Reply:
can you provide more information for why this does not work?